pages/index.js

```diff
@@ -98,6 +99,16 @@ export default () => (
 
     <SponsorsSection />
 
-    <JoinSection />
+    {!isLoggedIn && <JoinSection />}
```

Comment:
note that there are other locations that <JoinSection> is rendered.

I wonder if it's best to connect that thing to redux so we don't have to think about it when we add it to other future pages... 🤔